jenkins/BUILD

```diff
@@ -14,7 +11,7 @@ py_test(
         "//jobs",
         "//prow:config.yaml",
     ],
-    deps = ["@yaml//:yaml"],
```

Comment:
I have no idea why this change is here, but it seems like something my vim plugin is doing.

it still passes, so 🤷‍♂️